code_schemes/ws_correct_dataset.json

```diff
@@ -0,0 +1,298 @@
+{
+  "SchemeID": "Scheme-fffffc6e",
+  "Name": "WS - Correct Dataset",
+  "Version": "0.0.0.1",
+  "Codes": [
+    {
+      "CodeID": "code-9919ab80",
+      "CodeType": "Normal",
+      "DisplayText": "dadaab hh language",
```

Comment:
```suggestion
      "DisplayText": "dadaab household language",
```

Reply:
This is fine provided it's never ambiguous which project a household language message belongs to. Because each location has a different short-code, there should never be any ambiguity, so this is fine provided the labellers select the correct option here. Given that risk, and that each location is coded separately in Coda across all datasets, I actually think we should have two ws_correct_dataset schemes, one for each location, but potentially sharing codes where it makes sense. Feel free to bring this to slack for further discussion if needed

Reply:
Splitting this into location  ws_correct_dataset. It is easy for an RA to pick the wrong label splitting them reduces the risk. Thanks